### PR TITLE
Display autocomplete options in form's current language

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?185094

@millerdev 

cc @NoahCarnahan

### CODE REVIEW

File: src/javaRosa.js [hunk 1]
```diff
@@ -1477,6 +1477,10 @@ define([
                     }
                 }
             });
+
+            if (form) {
+                form.vellum.refreshCurrentMug();
```
Comment (orangejenny) [inline src/javaRosa.js]: Too heavy-handed?

Comment (orangejenny) [inline src/javaRosa.js]: this is causing all the test failures

Comment (millerdev) [inline src/javaRosa.js]: Oof... my first impression is yes. There should be a better way, but I'm not sure what problem this is solving so it's hard to suggest a better way without digging in deeper.

**Comment by orangejenny** [comment]
@millerdev different approach: fire & listen for an event on language change

**Comment by millerdev** [comment]
Excellent! I like this approach much better.

**Pull request merged**